builder/job.go

```diff
@@ -121,24 +124,52 @@ func Build(d *daemon.Daemon, buildConfig *Config) error {
 	} else if urlutil.IsURL(buildConfig.RemoteURL) {
 		f, err := httputils.Download(buildConfig.RemoteURL)
 		if err != nil {
-			return err
+			return fmt.Errorf("Error downloading remote context %s: %s\n", buildConfig.RemoteURL, err)
```

Comment:
please use `%v` for the err, and remove the `\n` unless you put it there for a reason?